mne/io/meas_info.py

```diff
@@ -60,8 +131,11 @@ class Info(dict):
     bads : list of str
         List of bad (noisy/broken) channels, by name. These channels will by
         default be ignored by many processing steps.
-    ch_names : list of str
+    ch_names : list-like of str (read-only)
```

Comment:
this one is tricky. It is actually a `_ChannelNameList`, but we don't want to expose private objects in the docs. It is an object that behaves like a list, but is read only. At any rate, it is not a numpy array.